sinonjs/sinon
# Export Sinon as an ES module

#### Purpose 
Implements #1805 by using Rollup to export the common js bundle. Enables Sinon to be used in `<script type="module>` blocks without polluting the global namespace.

#### How to verify - mandatory
1. Check out this branch
2. `npm link`
3. `cd /tmp; mkdir foo; cd foo; npm install esm; npm link sinon;`
3.  `node -r esm -e 'import sinon from "sinon"; console.log("sinon.stub()", sinon.stub().returns(42)())'`
4. Verify that it outputs: `sinon.stub() 42`

#### Checklist for author


One might wonder if there is a need for having both browserify and rollup, but IMHO it adds little maintenance weight. We could get rid of one of them, but perhaps not in this PR.
